src/dreamchecker/lib.rs

```diff
@@ -1702,10 +1702,13 @@ impl<'o, 's> AnalyzeProc<'o, 's> {
             },
 
             Term::Call(unscoped_name, args) => {
-                if unscoped_name == "sleep" || unscoped_name == "alert" || unscoped_name == "shell" || unscoped_name == "winexists" || unscoped_name == "winget" {
-                    if self.inside_newcontext == 0 {
+                if matches!(unscoped_name.as_str(),
+                    "sleep"
+                    | "alert"
+                    | "shell"
+                    | "winexists"
+                    | "winget" if self.inside_newcontext == 0) {
```

Comment:
style nit: the `if` guard doesn't depend on the pattern, so it should probably go outside the `matches!()` invocation

Reply:
true